openshift-setup/lagoon-builddeploy-controller.yml

```
@@ -213,7 +213,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: amazeeio/lagoon-builddeploy:v0.1.5
+        image: amazeeio/lagoon-builddeploy:v0.1.7
```

```suggestion
        image: amazeeio/lagoon-builddeploy:v0.1.5
```
My hunch is that the controller update is messing with this pr - is it needed for this? I'm testing the update separately...